lab-isaac/lib/read-file.js

```diff
@@ -4,26 +4,22 @@
 const vegies = module.exports = {};
 const fs = require('fs');
 
-vegies.read = (callback) => {
-  let first, second, third;
+vegies.read = () => {
+  // let first, second, third;
 
   fs.readFile(`${__dirname}/../assets/one.txt`, (err, data) => {
     if(err) console.error(err);
-    first = data.toString('hex', 0, 8);
-
-
+    let first = data.toString('hex', 0, 8);
+    console.log('1st ', first);
     fs.readFile(`${__dirname}/../assets/two.txt`, (err, data) => {
       if(err) console.error(err);
-
-      second = data.toString('hex', 0, 8);
-
-
+      let second = data.toString('hex', 0, 8);
+      console.log('2nd ', second);
       fs.readFile(`${__dirname}/../assets/three.txt`, (err, data) => {
         if(err) console.error(err);
-
-        third = data.toString('hex', 0, 8);
-        callback({first, second, third});
-
+        let third = data.toString('hex', 0, 8);
+        console.log('3rd ', third);
```

Comment:
You've got the right idea, so no need to refactor again, but you could optionally add each of these data points to an array scoped within the `read` method, and then return that array once the third nested callback completes.